shell.nix

```diff
@@ -0,0 +1,29 @@
+{ pkgs ? import <nixpkgs> {} }:
+
+let
+  dependencies = with pkgs; [
+    dotnetCorePackages.sdk_7_0
+    glfw
+    SDL2
+    glibc
```

Comment:
is it possible to force it to an older ver of glibc to avoid that one bug

Reply:
i do not use nixos. but i personally have never encountered That One Bug despite the warning, would be worth testing it first to see if it happens at all on nixos with this flake

Reply:
While I haven't ran into the funny bug with this, there's a few options here to make extra sure it won't happen:

- pinning a newer/older nixpkgs rev on the flake which ships a glibc version where the bug doesn't happen
- adding the environment variable that fixes the bug to the shell hook

The former is useless in case someone runs this with `nix-shell` as it bypasses the flake and just uses the system nixpkgs revision, so I'm going with the latter just in case.

Reply:
o yea i forgot about the env var lol